src/Components/NavbarHorizontal/PersonalTools.php

```diff
@@ -210,14 +210,20 @@ protected function getDropdownToggle(): string {
 
 		}
 
+		// TODO Rename '...LinkText' to '...LinkTitle' in both the hook and variable.
```

Comment:
We'll need to be careful with this. I have no idea how many people use this hook, but if we change the name then it would be a breaking change. I agree that your suggested naming makes more sense for what is actually being changed.

Reply:
Certainly... it is one of those housekeeping/code-cleaning things to save for the next "major release with breaking changes", along with removing deprecated options/functions, etc. and big bold warnings about the changes.  (That would also be a good time to introduce documentation for the hooks!)